### PR TITLE
Fix CauchyModel documentation

### PR DESCRIPTION
**Pull request by sofianehaddad**
Never too late

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -86,6 +86,7 @@
 
 === Bug fixes ===
  * #1260 (RandomWalkMetropolisHastings does not handle non-symmetric proposals properly)
+ * #1830 (Incompatible documentation for CauchyModel)
```
Comment (jschueller) [inline ChangeLog:89]: that's the 1.18 changelog, not the 1.19

Comment (sofianehaddad) [reply ChangeLog:89]: Indeed thanks

File: python/doc/theory/probabilistic_modeling/parametric_spectral_model.rst [hunk 2]
```diff
@@ -22,15 +22,16 @@ Parametric spectral density functions
 
 **The Cauchy spectral model**
 
-Its is associated to the Exponential covariance model. The Cauchy spectral model is defined by:
+Its is associated to the Kronecker covariance model built upon an exponential covariance model (AbsoluteExponential). The Cauchy spectral model is defined by:
```
Comment (josephmure) [inline python/doc/theory/probabilistic_modeling/parametric_spectral_model.rst:25]:
```suggestion
It is associated to the Kronecker covariance model built upon an exponential covariance model (AbsoluteExponential). The Cauchy spectral model is defined by:
```

File: python/doc/theory/probabilistic_modeling/parametric_spectral_model.rst [hunk 3]
```diff
@@ -22,15 +22,16 @@ Parametric spectral density functions
...
 .. math::
   :label: cauchyModel
 
-    S_{ij}(f) = \displaystyle \frac{4R_{ij}a_ia_j(\lambda_i+ \lambda_j)}{(\lambda_i+ \lambda_j)^2 + (4\pi f)^2}, \quad \forall (i,j) \leq d
+    S_{ij}(f) = \displaystyle 2 \Sigma_{ij}\prod_{k=1}^{n} \frac{\theta_k}{1 + (2\pi \theta_k f)^2}, \quad \forall (i,j) \leq d
 
-where :math:`\mat{R}`, :math:`\vect{a}` and :math:`\vect{\lambda}`
-are the parameters of the Exponential covariance model defined in
+where :math:`\mat{\Sigma}` is the covariance matrix of the Kronecker
+covariance model, :math:`\vect{\theta}` the scale parameters of
```
Comment (josephmure) [inline python/doc/theory/probabilistic_modeling/parametric_spectral_model.rst:33]:
```suggestion
covariance model and :math:`\vect{\theta} = (\theta_1, \dots, \theta_n)`
is the vector of scale parameters of
```

File: python/doc/theory/probabilistic_modeling/parametric_spectral_model.rst [hunk 4]
```diff
@@ -22,15 +22,16 @@ Parametric spectral density functions
...
 **The Cauchy spectral model**
 
-Its is associated to the Exponential covariance model. The Cauchy spectral model is defined by:
+Its is associated to the Kronecker covariance model built upon an exponential covariance model (AbsoluteExponential). The Cauchy spectral model is defined by:
 
 .. math::
   :label: cauchyModel
 
-    S_{ij}(f) = \displaystyle \frac{4R_{ij}a_ia_j(\lambda_i+ \lambda_j)}{(\lambda_i+ \lambda_j)^2 + (4\pi f)^2}, \quad \forall (i,j) \leq d
+    S_{ij}(f) = \displaystyle 2 \Sigma_{ij}\prod_{k=1}^{n} \frac{\theta_k}{1 + (2\pi \theta_k f)^2}, \quad \forall (i,j) \leq d
```
Comment (josephmure) [inline python/doc/theory/probabilistic_modeling/parametric_spectral_model.rst:30]:
```suggestion
    [S(f)]_{ij} = \displaystyle 2 \mat{\Sigma}_{ij}\prod_{k=1}^{n} \frac{\theta_k}{1 + (2\pi \theta_k f)^2}, \quad \forall (i,j) \leq d
```
I think this notation is clearer: S is defined in the introduction, but not S_{ij}. Also, Sigma needs to be consistently written as a matrix.

File: python/doc/theory/probabilistic_modeling/parametric_spectral_model.rst [hunk 5]
```diff
@@ -22,15 +22,16 @@ Parametric spectral density functions
...
 
-where :math:`\mat{R}`, :math:`\vect{a}` and :math:`\vect{\lambda}`
-are the parameters of the Exponential covariance model defined in
+where :math:`\mat{\Sigma}` is the covariance matrix of the Kronecker
+covariance model, :math:`\vect{\theta}` the scale parameters of
+the AbsoluteExponential covariance model defined in
 section [ParamStationaryCovarianceFunction]. The relation
 :eq:`cauchyModel` can be explained with the spatial covariance function
```
Comment (josephmure) [inline python/doc/theory/probabilistic_modeling/parametric_spectral_model.rst:36]: What does "can be explained" mean? "can be derived from" perhaps? In any case, it is not very informative.

**Review by josephmure — COMMENTED** [review]
Good that you took care of this. A few requests, plus questions that are more related to the doc page itself than to your changes.

File: python/doc/theory/probabilistic_modeling/parametric_spectral_model.rst (hunk 5 above)
Comment (sofianehaddad) [reply python/doc/theory/probabilistic_modeling/parametric_spectral_model.rst:37]: I think the original test refers to the way we define this spectral model. It was the Fourier counterpart of the oldest `ExponentialModel`. `derived` is indeed more informative

File: python/doc/theory/probabilistic_modeling/parametric_spectral_model.rst [hunk 6]
```diff
@@ -22,15 +22,16 @@ Parametric spectral density functions
...
 
-    S_{ij}(f) = \displaystyle \frac{4R_{ij}a_ia_j(\lambda_i+ \lambda_j)}{(\lambda_i+ \lambda_j)^2 + (4\pi f)^2}, \quad \forall (i,j) \leq d
+    S_{ij}(f) = \displaystyle 2 \Sigma_{ij}\prod_{k=1}^{n} \frac{\theta_k}{1 + (2\pi \theta_k f)^2}, \quad \forall (i,j) \leq d
 
-where :math:`\mat{R}`, :math:`\vect{a}` and :math:`\vect{\lambda}`
-are the parameters of the Exponential covariance model defined in
+where :math:`\mat{\Sigma}` is the covariance matrix of the Kronecker
+covariance model, :math:`\vect{\theta}` the scale parameters of
+the AbsoluteExponential covariance model defined in
 section [ParamStationaryCovarianceFunction]. The relation
```
[the remark this answers is not in the document]
Comment (sofianehaddad) [reply python/doc/theory/probabilistic_modeling/parametric_spectral_model.rst:36]: Good question. I will drop this

**Review by josephmure — APPROVED** [review]

**Comment by sofianehaddad** [comment]
I notice here that similarly to #2019 this "theory" doc is in fact a use case.
```
  ...
  This use case illustrates how the User can create a density spectral
  function from parametric models. The library proposes the *Cauchy
  spectral model* as a parametric model for the spectral density
  function :math:`S`.
  ...
```
For now I will at least fix both doc (models equation, notation). We will see further how to deal with the presence of use cases in the `theory` part

**Review by regislebrun — APPROVED** [review]

**Pull request merged**